tests/e2e_tests/l2b_to_distortion_e2e.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import os
+import numpy as np
+from corgidrp.data import AstrometricCalibration
+
+import corgidrp.mocks as mocks
+import corgidrp.astrom as astrom
+import corgidrp.walker as walker
+import pytest
+import glob
+
+
+thisfile_dir = os.path.dirname(__file__) # this file's folder
+
+@pytest.mark.e2e
+def test_l2b_to_distortion(e2eoutput_path):
```

Comment:
all of our e2e test functions has e2e_data_path as one of the inputs. If you don't use it, that's fine, but I want to keep the structure the same.